index.js

```diff
@@ -422,6 +422,8 @@ class Raspi extends EventEmitter {
     const pinInstance = this[getPinInstance](this.normalize(pin));
     if (pinInstance.mode === INPUT_MODE && value === HIGH) {
       this[pinMode]({ pin, mode: INPUT_MODE, pullResistor: PULL_UP });
+    } else if (pinInstance.mode === INPUT_MODE && value === LOW) {
```

Comment:
@nebrius I'm ok with the minor change in Johnny-Five, since it's really just making a way to explicitly do something that is the default behavior of StandardFirmata.